Public/Src/Cache/DistributedCache.Host/Service/Internal/DistributedContentStoreFactory.cs

```diff
@@ -451,5 +448,43 @@ public KeyVaultConfigurationException(string message, Exception innerException)
             {
             }
         }
+
+        private static RetryPolicy CreateKeyVaultRetryPolicy()
+        {
+            return new RetryPolicy(
+                new KeyVaultRetryPolicy(),
+                new ExponentialBackoff(
+                    name: "KeyVaultExponentialBackoff",
+                    retryCount: 5,
+                    minBackoff: TimeSpan.FromSeconds(10),
+                    maxBackoff: TimeSpan.FromSeconds(60),
+                    deltaBackoff: TimeSpan.FromSeconds(10),

```

Comment:
>deltaBackoff [](start = 20, length = 12)

NIT: pr is good, just a generic comment that api makes it a bit hard to read what is really happening from the callsite without knowing the implementation.